src/styles/_vars.scss

```diff
@@ -3,4 +3,5 @@ $engine_assets: "./assets" !default;
 $engine_even_color: #f7f7f7 !default;
 $engine_loading_image: url('#{$engine_assets}/loading.svg') !default;
 $engine_loading_static_image: url('#{$engine_assets}/loading_s.svg') !default;
-$engine_hover_outline_color: 2px solid orange !default;
+$engine_hover_outline_width: 2px;
```

Comment:
should have the `!default` flag at the end

Reply:
I simply forgot that. -> Fixed.